recipe/meta.yaml

```diff
@@ -40,11 +40,13 @@ outputs:
         - {{ compiler("c") }}
         - {{ compiler("cxx") }}
         - {{ compiler("cuda") }}         # [cuda_compiler_version != "None"]
-        - {{ cdt("libnl") }}
-        - {{ cdt("libibcm-devel") }}
-        - {{ cdt("libibverbs-devel") }}
-        - {{ cdt("librdmacm-devel") }}
+        - {{ cdt("libnl") }}             # [cdt_name == "cos6"]
+        - {{ cdt("libnl3") }}            # [cdt_name == "cos7"]
```

Comment:
Turns out the RDMA CDTs link against `libnl3` on CentOS 7 (not `libel`) and that wasn't being found. Interestingly the `yum` packages don't list this dependency (maybe this is because it is assumed to always be there? it lives in `/lib64` instead of `/usr/lib64); however, it is needed. So adding this fixed it.